packages/design-tokens/src/tokens/functional/components/river/base.json

```diff
@@ -1,32 +1,39 @@
 {
   "brand": {
     "River": {
-      "layout": {
+      "small": {
+        "padding": {
```

Comment:
More a question, but this feels like implementation detail that could become semantically incorrect if we refactored to use; say margin instead of padding in future. 

What do you think about using `inner` and `outer` spacing as the descriptors? so 

E.g. 

```diff
- --brand-River-small-padding
- --brand-River-small-gap
+ --brand-River-small-spacing-outer
+ --brand-River-small-spacing-inner
```

Reply:
Love the idea ❤️